app/Models/Organization.php

```diff
@@ -298,13 +301,57 @@ public function hasAddedDetails(): bool
 
     public function isPublishable(): bool
     {
-        return ! is_null($this->languages)
-            && ! is_null($this->working_languages)
-            && ! is_null($this->locality)
-            && ! is_null($this->region)
-            && ! is_null($this->about)
-            && ! is_null($this->service_areas)
-            && ! is_null($this->contact_person_name);
+        $publishRules = [
+            'about.en' => 'required_without:about.fr',
+            'about.fr' => 'required_without:about.en',
+            'consulting_services' => [
+                Rule::requiredIf(fn () => $this->isConsultant()),
+                Rule::excludeIf(fn () => ! $this->isConsultant()),
+            ],
+            'contact_person_email' => 'nullable|required_without:contact_person_phone',
+            'contact_person_name' => 'required',
+            'contact_person_phone' => 'nullable|required_if:contact_person_vrs,true|required_without:contact_person_email',
+            'extra_attributes.has_age_brackets' => 'required',
+            'extra_attributes.has_ethnoracial_identities' => 'required',
+            'extra_attributes.has_gender_and_sexual_identities' => 'required',
+            'extra_attributes.has_refugee_and_immigrant_constituency' => 'required',
+            'extra_attributes.has_indigenous_identities' => 'required',
+            'languages' => 'required',
+            'locality' => 'required',
+            'name.en' => 'required_without:name.fr',
+            'name.fr' => 'required_without:name.en',
+            'preferred_contact_method' => 'required',
+            'region' => 'required',
+            'roles' => 'required',
+            'service_areas' => 'required',
+            'staff_lived_experience' => 'required',
+            'type' => 'required',
+            'working_languages' => 'required',
+        ];
+
+        try {
+            Validator::validate($this->toArray(), $publishRules);
+        } catch (ValidationException $exception) {
+            return false;
+        }
+
+        if (! $this->livedExperiences()->count()) {
+            return false;
+        }
+
+        if (! $this->areaTypes()->count()) {
+            return false;
+        }
+
+        if ($this->extra_attributes['has_indigenous_identities'] && ! $this->indigenousIdentities()->count()) {
+            return false;
+        }
+
+        if ($this->extra_attributes['has_age_brackets'] && ! $this->ageBrackets()->count()) {
+            return false;
+        }
```

Comment:
I think there needs to be additional checks for has_ethnoracial_identities and has_gender_and_sexual_identities— if either of these are true, the related identities need to be attached (the way you've done with indigenous identities and age brackets).